### PR TITLE
[OBSDEF-13994] CSS changes for Datalist component

### PR DESCRIPTION
**Pull request by psivakumaran**
Summary of the changes
OBSDEF-13994

- Added stories for datalist
- css changes according to the dell standards

Testing link - http://192.168.2.128:3000/#/objectscale
Token
>
eyJraWQiOiI2NUE1QjgwNkEwMkEzODMyQTNEQzI5QzA0RTUyMzQxMkRFNTU4NzZEIiwiYWxnIjoiUlMyNTYifQ.eyJzdWIiOiJBZG1pbmlzdHJhdG9yQGFuZHJlLmxvY2FsIiwiYXVkIjoidm13YXJlLXRlczp2Yzp2bnM6azhzIiwiZG9tYWluIjoiYW5kcmUubG9jYWwiLCJpc3MiOiJodHRwczpcL1wvdmNlbnRlcjcwLWFuZHJlLmVjcy5ob3AubGFiLmVtYy5jb21cL29wZW5pZGNvbm5lY3RcL2FuZHJlLmxvY2FsIiwiZ3JvdXBfbmFtZXMiOlsiU3lzdGVtQ29uZmlndXJhdGlvbi5BZG1pbmlzdHJhdG9yc0BhbmRyZS5sb2NhbCIsIlVzZXJzQGFuZHJlLmxvY2FsIiwiQ0FBZG1pbnNAYW5kcmUubG9jYWwiLCJFdmVyeW9uZUBhbmRyZS5sb2NhbCIsIkxpY2Vuc2VTZXJ2aWNlLkFkbWluaXN0cmF0b3JzQGFuZHJlLmxvY2FsIiwiU3lzdGVtQ29uZmlndXJhdGlvbi5SZWFkT25seUBhbmRyZS5sb2NhbCIsIlN5c3RlbUNvbmZpZ3VyYXRpb24uU3VwcG9ydFVzZXJzQGFuZHJlLmxvY2FsIiwiU3lzdGVtQ29uZmlndXJhdGlvbi5CYXNoU2hlbGxBZG1pbmlzdHJhdG9yc0BhbmRyZS5sb2NhbCIsIkFkbWluaXN0cmF0b3JzQGFuZHJlLmxvY2FsIl0sImV4cCI6MTY0MzY1MDk5MSwiaWF0IjoxNjQzNjE0OTkxLCJqdGkiOiIwMGQxNzAzNy0yY2I1LTQwNzMtOWNkYS0zMzRjYmI0N2RiYzgiLCJ1c2VybmFtZSI6IkFkbWluaXN0cmF0b3IifQ.sjFXf-AASsIBhkoc3CzFQzOQvawaP5xMXfsSlQkCLbY-xE8nRMrKdCrnqjf46hcS9LT8dk-LRMchOH7GLQeklo61C-Pn0Bj22tuaGcWerATf3meQWZ9w9rVu5PcTeJ4ecycM8VJxjJg_MBNxZwuFVz0bSPQUSKftEKDZdJOyFTmyJuHFaJ-iX6a4Rp6JGMHorSOpe48GbfxagYD1vImwX7ad4fFcG3C_OUpfY9NAutk3kL7vz-zmaOSYfGJwo89w2tanw_r-EMjFYurG1VVLXlaqCC69Vdd8A7gtBorgWrU5sZrD4It1KazDJU7po5_ihfBkyr_Y4PuexSgwoMPVxg

### CODE REVIEW

File: src/components/datalist/Datalist.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+        <div>
+            <DataList
+                placeHolder="No label"
+                helperText="Select any option or create one"
+                label="Select Item"
+                style={{width: "50%"}}
+            >
+                <DataListOption value="Item1" />
+                <DataListOption value="Item2" />
+                <DataListOption value="Item3" />
```
Comment (akate1) [inline src/components/datalist/Datalist.stories.tsx:70]: can we create constant for options ?

File: src/styles/components/DataList.scss [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+ *     http://www.apache.org/licenses/LICENSE-2.0
+*/
+
+@import "../common/mixins";
+@import "../common/colors";
+@import "../common/fonts";
+@import "../common/borders";
+@import "../common/layout";
+
+.clr-control-container {
```
Comment (akate1) [inline src/styles/components/DataList.scss:17]: This CSS class is present in other CSS files as well like input, label, button etc. Can we create mixin for this ?

**Review by ShitalSD — APPROVED** [review]

File: src/styles/common/_mixins.scss [hunk 3]
```diff
@@ -115,3 +116,22 @@
...
+    border-style: solid;
+    border-width: $border-default; 
+}
+
+@mixin input-focus-style {
+    outline: none !important;
+    border-radius: $input-focus-border-radius !important;
+    border: $border-default solid $input-focus-border-color;
+    background-size: 0 !important;
+}
```
Comment (singha158ecs) [inline src/styles/common/_mixins.scss:137]: Please add a new line here.

**Review by singha158ecs — APPROVED** [review]
Check if we can remove bottom border of datalist on focus/active.
Rest LGTM.

**Pull request merged**